vaadin-grid-flow-parent/vaadin-grid-flow-integration-tests/src/test/java/com/vaadin/flow/component/treegrid/it/TreeGridPreloadIT.java

```diff
@@ -94,29 +86,7 @@ public void firstExpanded_shouldPreLoadDataForExpandedChildren() {
     @Test
     public void firstExpanded_shouldOptimizeDataProviderFetches() {
         open(Arrays.asList(0), null, null, null, null);
-        Assert.assertEquals("55", dataProviderFetchCount.getValue());
-    }
-
-    @Test
-    public void firstExpanded_shouldNotHaveDataForExpandedRootItemsOutsideViewportEstimate() {
```

Comment:
The removed tests in this file relied on `setParentRequestedRange`, which is no longer available